google_gke_infra/README.md

```diff
@@ -1,56 +1,58 @@
 # GKE Infra
 
-This Terraform module provisions a regional `GKE cluster`, `VPC`, and `Subnet`. Optionally, it can be configured to create a service account with limited permissions for the K8s Nodes if no `service_account` value is provided (see [Use Least Privilege Service Accounts for your Nodes](https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster#use_least_privilege_sa)).
+This Terraform module provisions a regional Google Kubernetes Engine (GKE) cluster, Virtual Private Cloud (VPC), and Subnet. Optionally, you can configure it to create a service account with limited permissions for the Kubernetes Nodes if no value is provided for the **service_account**. For details, see [Use least privilege service accounts for your Nodes](https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster#use_least_privilege_sa).
```

Comment:
K8s> Kubernetes: https://kyma-project.io/community/guidelines/content/#style-and-terminology-style-and-terminology-capitalized-terms